package Chapter8;

import java.io.IOException;

import Chapter4.cashCard;

public class test {

	public static void main(String[] args) throws IOException {
		CashCard c = new CashCard("1", 500, 1);
		try {
			c.charge(-200);
			System.out.println(c.getBalance());
		} catch (InsufficientException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}
		
	}

}
